File: java-main/java-main/Nestedforloop/p12.java
class Demo{

	public static void main(String[]args){

		int N=4;

		for(int i=1;i<=4;i++){
			for(int j=1;j<=4;j++){
				if(j==1 || j==N){
					System.out.print("*");
				}else{
					System.out.print("  ");
				}
			}
			System.out.println();
		}
	}
}

File: flutter-main/flutter-main/Inheritence/p12.dart
mixin DemoParent{
	void m1(){
		print("In m1");
	}
}
class Demo{
	void m2(){
		print("In m2");
	}
}
class Demochild extends Demo with DemoParent{

}
void main(){
	Demochild obj = new Demochild();
	obj.m1();
	obj.m2();
}
